releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -3,7 +3,11 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
 ### v5.0.0 - February, 02 2023
-Updated the task to require Node.js 16. It previously required 10
+**Breaking Changes**
+  - Updated the task to require Node.js 16. It previously required 10.
+  - Updated the minimum supported azure pipeline agent version to [2.206.1](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.206.1) or later. Previously it was [2.144.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.144.0).
+  - Importing key vault references value as an escaped json strings is not supported, please specify the key vault reference value as a json object.

```

Comment:
This is a change for all files using application/json content type right, not just kv set? If so, we'll want to call that out.

Also, may help to provide a snippet of what has changed.

Something like

Before when you had this

```
bla bla
```

you got this
```
bla bla bla
```

Now, the input needs to be 
```
bla bla bla
```

Reply:
It only applies for default content profile, using application/json content type. 
How about something like :
"Importing configurations with values as escaped json strings and the content type "application/json" is no longer supported. Please specify value as json object."

**Before**
 Sample configuration file
```
{  
  "app": "{\"uri\":\"https://keyvault.vault.azure.net/secrets/secret\"}"
}
```
Results:

| Key           | Value         |
| ----------- | ----------- |
| app           |   "{"uri":"https://keyvault.vault.azure.net/secrets/secret"}"        |

**After**
The input should be

```
{  
  "app": {
      "uri": "https://keyvault.vault.azure.net/secrets/secret " 
     }
}
```

Reply:
I was thinking since the breaking change is in relation to importing configurations, we can  show the before and after of the snipped that has changed and skip showing the results

Reply:
>How about something like :
"Importing configurations with values as escaped json strings and the content type "application/json" is no longer supported. Please specify value as a json object."


I would avoid the phrase "is no longer supported." It's not that we removed support but rather we changed the behavior. We now convert the values of properties in the settings to json strings.

> I was thinking since the breaking change is in relation to importing configurations, we can show the before and after of the snipped that has changed and skip showing the results

Sounds reasonable. But, I would include the content type that goes with the associated input snippet

Reply:
> I would avoid the phrase "is no longer supported." It's not that we removed support but rather we changed the behavior. We now convert the values of properties in the settings to json strings.

Yes good point.

Reply:
This has been updated thanks